.github/CODEOWNERS

```diff
@@ -34,73 +29,40 @@
 
 
 # ================================================
-#  GitHub username registry
-#  (just to make this file easier to understand)
+# All the files in the repo.
 # ================================================
 
-#  Platonn - Krzysztof Platis
-#  znikola - Nikola Zarić
-#  developpeurweb - Miguel Estrada
-#  psmul - Patryk Smul
-#  plabadie - Patrick Labadie
-#  WeizhengSap - Weizheng Gao
-#  hackergil - Gilberto Alvarado Jimenez
+* @SAP/spartacus-codeowners
 
-######################################################################################################
-#
-# CODEOWNERS rules
-# -----------------
-#
-# All the following rules are applied in the order specified in this file.
-# The last rule that matches wins!
-#
-# See https://git-scm.com/docs/gitignore#_pattern_format for pattern syntax docs.
-#
-######################################################################################################
 
 # ================================================
-#  Default Owners
-# (in case no pattern matches a path in a PR - this should be treated as a bug and result in adding the path to CODEOWNERS)
+#  This code owner file
 # ================================================
-
-# * @SAP/spartacus-codeowners
+/.github/CODEOWNERS          @SAP/spartacus-admins
 
 # ================================================
-#  Core
+#  Folders where anyone with write access to the repo can approve Pull Requests
 # ================================================
-/projects/core/**                                   @SAP/spartacus-codeowners
-/projects/storefrontlib/**                          @SAP/spartacus-codeowners
-/feature-libs/**                                    @SAP/spartacus-codeowners
-/core-libs/**                                       @SAP/spartacus-codeowners
-/ci-scripts/**                                      @SAP/spartacus-codeowners
-/scripts/**                                         @SAP/spartacus-codeowners
-/tools/**                                           @SAP/spartacus-codeowners
-/*                                                  @SAP/spartacus-codeowners
 
+/doc/          @SAP/spartacus-project-team
 
-# ================================================
-#  Styles
-# ================================================
-/projects/storefrontstyles/**                       @SAP/spartacus-codeowners
+/projects/storefrontapp-e2e-cypress/          @SAP/spartacus-project-team
```

Comment:
Discussion needed here perhaps.  We should allow teams to at least approve their own e2e test code.
We also want a low maintenance code owner config.
I can think of 2 options:
a) Open the e2e test project to _all_ contributors with write access in the repo.
b) Refactor the e2e to make sure that e2e tests for a given feature/domain are all under a sub-folder with a consistent name.  For example, `asm` related files could all be under multiple `asm` subfolders. (they already are in most places).  Then we can apply this kind of code owner config: 
```
/projects/storefrontapp-e2e-cypress/**/asm/            @SAP/spartacus-codeowners @SAP/spartacus-iceman
```